mycroft/skills/phillips_hue/vocab/en-us/ActivateSceneKeyword.voc

```diff
@@ -0,0 +1,6 @@
+activate scene
+activate seen
+set my lights to
+set scene
+set seen
```

Comment:
Oh, alright. Still, I think rather than building around flaws in the STT responses, we could try to control them more. Try changing the vocab to something like `set the scene` and `activate the scene` so that seen will be thought of as a less grammatical alternative.